src/main/target/common_defaults_post.h

```diff
@@ -329,15 +336,19 @@
 #endif
 
 #if !defined(GYRO_1_ALIGN)
-#define GYRO_1_ALIGN            ALIGN_DEFAULT
```

Comment:
Not too sure about this - to my understanding this will change the gyro alignment for existing targets if `ALIGN_DEFAULT` for the gyro used is not `CW0_DEG`, which we should avoid.

In a broader discussion, I do not know what the value of `ALIGN_DEFAULT` is, especially when moving to Unified Targets, so we should probably investigate removing it after 4.1.

Reply:
it's ok, the default ends up being `CW0_DEG` when `ALIGN_DEFAULT` is used.  This can be seen in master in the `alignSensors` `switch()` handler for `default:`.   This is why ALIGN_DEFAULT is removed in the other PR.